default/species.txt

```diff
@@ -697,12 +697,25 @@ Species
         [[AVERAGE_HAPPINESS]]
         [[GREAT_SUPPLY]]
         [[AVERAGE_DEFENSE_TROOPS]]
-     
+
         // not for description
         [[AVERAGE_PLANETARY_SHIELDS]]
         [[AVERAGE_PLANETARY_DEFENSE]]
         [[LARGE_PLANET]]
         [[BROAD_EP]]
+
+        EffectsGroup
+            scope = And [
+                Planet
+                Species name = "SP_GEORGE"
+                Not OwnedBy empire = Source.Owner   // would be redundant to re-assign ownership to own planets
```

Comment:
Just figured, you mean "redundant to grant visibility to your own planets" here, right? Ownership isn't changing, this is the list of criteria for things that become visible.